llvm/test/tools/llvm-readobj/XCOFF/loader-section-relocation.test

```diff
@@ -76,10 +80,12 @@ Sections:
 # COMMON-NEXT:     Loader Section Relocations {
 # NOEXPAND64-NEXT:           Vaddr            Type        SecNum  SymbolName (Index)
 # NOEXPAND64-NEXT:     0x0000000020000294 0x1f00 (R_POS)       2    .data (1)
-# NOEXPAND64-NEXT:     0x000000002000029c 0x1f00 (R_POS)       2    myintvar (3)
+# NODEMAN64-NEXT:     0x000000002000029c 0x1f00 (R_POS)       2    _Z4funcv (3)
```

Comment:
As whitespace isn't significant, please could you realign all the patterms so that they line up vertically. Applies throughout this PR.